cmd/roomservice/main.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * roomservice
+ *
+ * An implementation of joinable rooms
+ *
+ * API version: 0.0.1
+ * Contact: decline@umass.edu
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/Galactus-Player/roomservice/galactuslib"
+)
+
+func main() {
+	log.Printf("Server started")
```

Comment:
Maybe put this log after the server has started (line 27) or change the message to "Server starting" instead?